assets/config/tailwind.config.js

```diff
@@ -83,6 +83,10 @@ module.exports = {
                     "Open Sans",
                     ...defaultTheme.fontFamily.sans,
                 ],
+                mono: [
+                    "Source Code Pro",
+                    ...defaultTheme.fontFamily.mono,
+                ]
```

Comment:
Just curious: Why the change to use Source Code Pro?

Reply:
Ah, I should've called this out in the PR. Mainly I just wanted to try a code font that was a tiny bit smaller at the same relative size (since the text in these new docs is a bit smaller, too), and our current font's always felt slightly to me. Source Code Pro is just one that I've used before and happen to like. Totally open to alternatives, though.